### ISSUE TITLE
knock latest has no commands found (worked in v0.1.8)

### ISSUE THREAD

**Issue by jshirley** [knocklabs/knock-cli#333]
Hey folks, recent versions of the knock cli seem to be packaged wrong or require additional steps to configure that aren't documented. Following the instructions from https://docs.knock.app/cli, and I get the same results from Homebrew.

```
$ npm install -g @knocklabs/cli@0.1.8$
$ knock whoami
‣ Loading... done
‣ Successfully verified the provided service token:
    Account name: <redacted>
    Service token name: <redacted>
```

However, after uninstalling and installing latest:
```
➤ knock
Knock CLI

VERSION
  @knocklabs/cli/0.1.13 darwin-arm64 node-v21.7.2

USAGE
  $ knock [COMMAND]

TOPICS
  plugins  List installed plugins.

COMMANDS
  help     Display help for knock.
  plugins  List installed plugins.

➤ knock whoami
 ›   Error: command whoami not found
```

**Comment by thomaswhyyou** [comment]
@jshirley Thank you for reporting, will take a look and follow up.

**Comment by thomaswhyyou** [comment]
@jshirley Hi there, this issue should be fixed in v0.1.14 - would you mind testing it out? Sorry for the trouble here, and thanks for reporting. Pls let us know if any other issues!

**Comment by jshirley** [comment]
Sorry Thomas, didn't see the update -- confirmed working for me, thank you!
